src/Dropdown/Dropdown.js

```diff
@@ -434,6 +434,7 @@ export class Dropdown extends React.PureComponent {
           }
         );
       } else if (this.state.active) {
```

Comment:
Can you add the  `&& (e.key === "ArrowDown" || e.key === "ArrowUp" || e.key === "Enter")` check to this line as well? Otherwise we'll be preventing the default action for all key presses, not only the ones we're monitoring. Either that or move that conditional outside line 413.

Reply:
ok